near-workspaces/__tests__/base.ava.ts

```diff
@@ -0,0 +1,112 @@
+import { Workspace } from 'near-workspaces-ava'
+import { NFTContractExtra, NFTContractMetadata } from '../../as/cNFT/assembly/models/persistent_nft_contract_metadata';
+import { CONTRACT_EXTRA, CONTRACT_METADATA } from '../utils/dummyData';
+import { call_accept_bid, call_burn_design, call_init, call_mint, call_nft_transfer, call_remove_bid, call_set_bid } from '../utils/functions';
+
+const log = (m)=>console.log("base.ava.ts: "+m);
+
+
+const workspace = Workspace.init(async ({ root }) => {
```

Comment:
To clean up some lines of code:
https://github.com/near/workspaces-js/blob/main/__tests__/01.basic-transactions.ava.ts#L12